### PR TITLE
Added name

### PR DESCRIPTION
**Pull request by sumitmukharjeeeeee**
Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/270"></a>

### CODE REVIEW

File: public/data/sumitmukharjeeeeee.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "links": [{
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/sumitmukharjeeeeee",
+        "icon": "github"
+    }, {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/SumitMukharjii",
+        "icon": "twitter"
+    }
+]
```
Comment (CodingSpecies) [inline public/data/sumitmukharjeeeeee.json:14]: Please can you format line 14, so the links closing square bracket is on line 13 @sumitmukharjeeeeee? I will check incase I am wrong though. Thanks

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/sumitmukharjeeeeee.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Sumit Mukharjee",
+    "bio": "Learning and Exploring",
+    "avatar": "https://github.com/sumitmukharjeeeeee.png",
+    "links": [{
```
Comment (CodingSpecies) [inline public/data/sumitmukharjeeeeee.json:5]: Please move the brace `{` to the next line, and format it please @sumitmukharjeeeeee

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/sumitmukharjeeeeee.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/sumitmukharjeeeeee",
+        "icon": "github"
+    }, {
```
Comment (CodingSpecies) [inline public/data/sumitmukharjeeeeee.json]: Please move the next curly braces after the comma, to the next line.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Comment by sumitmukharjeeeeee** [comment]
Please have a look now

File: public/data/sumitmukharjeeeeee.json [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
+      "url": "https://github.com/sumitmukharjeeeeee",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SumitMukharjii",
+      "icon": "twitter"
     }
- ]
+  ]
```
Comment (CodingSpecies) [inline public/data/sumitmukharjeeeeee.json:16]: @sumitmukharjeeeeee sorry, now it is correctly formatted, that can be moved back a space. Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/sumitmukharjeeeeee.json (hunk 3 above)
Comment (CodingSpecies) [reply public/data/sumitmukharjeeeeee.json]: This is complete. ✅

File: public/data/sumitmukharjeeeeee.json (hunk 4 above)
Comment (sumitmukharjeeeeee) [reply public/data/sumitmukharjeeeeee.json:16]: Is it correct now?

Comment (CodingSpecies) [reply public/data/sumitmukharjeeeeee.json:16]: it is all good, but the square bracket needs moving back. thank you.

Comment (Panquesito7) [reply public/data/sumitmukharjeeeeee.json:16]: For me, this is already fine. It is correct as-is.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

File: public/data/sumitmukharjeeeeee.json (hunk 3 above)
Comment (krishdevdb) [reply public/data/sumitmukharjeeeeee.json]: We can click the resolve button after the issues are fixed 🙂

File: public/data/sumitmukharjeeeeee.json (hunk 4 above)
Comment (krishdevdb) [reply public/data/sumitmukharjeeeeee.json:16]: RE: We can click the resolve button after the issues are fixed 🙂

**Review by krishdevdb — APPROVED** [review]

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

**Review by schmelto — APPROVED** [review]

**Pull request merged**